.vscode/tasks.json

```diff
@@ -0,0 +1,10 @@
+{
+    "version": "2.0.0",
+    "tasks": [{
+        "label": "serve",
+        "type": "shell",
+        "command": "webpack serve --env SUPPRESS_OPEN --config config/webpack.dev.js",
+        "problemMatcher": [],
+        "isBackground": true,
+    }]
+}
```

Comment:
I started with an npm task, but converted to type `shell` specifically because I wanted to pass `SUPPRESS_OPEN` and not update that in the package scripts.  If you think we're good just never opening, I can go back to using npm tasks.

The other thing is - we **need** `isBackground` set to true, so I can't take yours wholesale.  That's because `start` watches changes and never actually returns an exit code, so the launch debugger never runs because it's waiting for the completion of the task.  See [Running preLaunchTask with watch mode in VS Code](https://stackoverflow.com/q/42672037/1366033)

Only reason the task is here is because it's needed as a `preLaunchTask` in the launch config.  Otherwise when hitting debug, you'd need to have already started a webpack serve manually for it to work.